examples/models/llama2/export_llama_lib.py

```diff
@@ -46,6 +46,7 @@
 logging.basicConfig(level=logging.INFO, format=FORMAT)
 
 pkg_name = __name__
+verbosity_setting = None
```

Comment:
Is there any way for a user to override this, or is it effectively hard-disabled for now?